sovrin_client/cli/helper.py

```diff
@@ -29,7 +29,7 @@ def getNewClientGrams():
         ADD_GENESIS_FORMATTED_REG_EX,
         # Regex for `new client steward with identifier <nym>`
         CLIENT_GRAMS_CLIENT_WITH_IDENTIFIER_FORMATTED_REG_EX,
-        # Regex for `client steward add sponsor bob` or `client steward
+        # Regex for `client steward add trustAnchor bob` or `client steward
```

Comment:
I'm not sure about the token "trustAnchor" in CLI. Is the regex case-sensitive? (I hope not)